src/common/verbose.cpp

```diff
@@ -1634,6 +1634,8 @@ std::string init_info_sdpa(const engine_t *e, const pd_t *pd) {
             ss << "mul:";
         ss << dnnl_dt2str(desc->scale_dt);
     }
+    ss << "acc:" << dnnl_dt2str(desc->kq_acc_dt) << ","
+       << dnnl_dt2str(desc->vs_acc_dt);
```

Comment:
Is this change aligned with verbose support work going along?

Reply:
The proposed verbose work doesn't include this information at all. I've dropped these lines for now.